openfga.yaml

```diff
@@ -19,12 +19,21 @@ pipeline:
         -X github.com/openfga/openfga/internal/build.Version=v${{package.version}}
         -X github.com/openfga/openfga/internal/build.Commit=$(git rev-parse HEAD)
         -X github.com/openfga/openfga/internal/build.Date=$(TZ="UTC" date +"%Y-%m-%dT%TZ")
+        -X github.com/openfga/openfga/internal/build.ProjectName=${{package.name}}
       output: openfga
       packages: ./cmd/openfga
 
   - uses: strip
```

Comment:
given we use go/build pipeline, we can omit this. go/build pipeline already does the stripping.